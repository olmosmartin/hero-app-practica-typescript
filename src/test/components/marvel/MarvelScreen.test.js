import { shallow } from "enzyme";
import '@testing-library/jest-dom';
import { MarvelScreen } from "../../../components/marvel/MarvelScreen";


describe('pruebas en MarvelScreen.tsx', () => {

    test('debe mostrarse correctamente con el snapshot', () => {
        const wrapper = shallow(<MarvelScreen />);
        expect(wrapper).toMatchSnapshot();
    })

})