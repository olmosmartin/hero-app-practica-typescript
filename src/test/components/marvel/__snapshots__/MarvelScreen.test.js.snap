// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pruebas en MarvelScreen.tsx debe mostrarse correctamente con el snapshot 1`] = `
<div>
  <h1>
    MarvelScreen
  </h1>
  <HeroList
    publisher="Marvel Comics"
  />
</div>
`;
